fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/jbatch/bulkdata/load/ChunkReader.java

```diff
@@ -179,7 +191,7 @@ public void open(Serializable checkpoint) throws Exception {
                 logger.info("open: Set tenant to default!");
             }
             if (fhirDatastoreId == null) {
-                fhirDatastoreId = Constants.DEFAULT_FHIR_TENANT;
+                fhirDatastoreId = DEFAULT_FHIR_TENANT;
```

Comment:
I know that this is "default" and will always be "default"...but still made me blink.

Reply:
It's a good question... I think Albert ran this as part of a CLI to test it.  I'm not sure this is needed when used in our framework.  It won't always be default, but we have logic to switch between it.